### PR TITLE
Add section on multisig deposits

### LINKED ISSUE #568: Document deposit for multi-sig
There was a lot of discussion in Kusama Direction about deposits for using multi-sig accounts. This should be documented and explained in the proxy section.

### PR DESCRIPTION
**Pull request by lsaether**
closes #568

### CODE REVIEW

File: docs/learn-accounts.md [hunk 1]
```diff
@@ -187,30 +187,93 @@ if the entities designated as the recipients come together in a new multi-sig un
...
+
+|               | Polkadot     | Kusama          |
+| ------------- | ------------ | --------------- |
+| DepositBase   | 200880000000 | 3347999999941.4 |
+| DepositFactor | 320000000    | 5333333312      |
+
+Let's consider an example of a multi-sig on Polkadot with a threshold of 2 and 3 signers: Alice,
+Bob, and Charlie. First Alice will create the call on chain by calling `as_multi` with the raw call.
+When doing this Alice will have to deposit 0.20152 DOTs while she waits for either Bob or Charlie to
+also approve the call. When Bob come to approve the call and execute the transaction, he will not
```
Comment (Swader) [inline docs/learn-accounts.md]:
```suggestion
also approve the call. When Bob comes to approve the call and execute the transaction, he will not
```

**Review by Swader — CHANGES_REQUESTED** [review]
We should clarify where the deposit is coming from: should the multisig itself have the money to cover the deposit, or is it the initiating signatory?

**Review by Swader — COMMENTED** [review]
If the script is the same as in your msig-utils script, something's off, it doesn't give the same address as the Polkadot JS Apps UI. Recommend fixing first.

**Comment by lsaether** [comment]
msig-util has been fixed and the code example points to that repository now. The original example had a bug in how it sorted the addresses before hashing.

**Pull request merged**